CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-cmake_minimum_required(VERSION 3.25)
```

Comment:
This will need some investigation; cmake versions also change the semantics of functions, so we need to make sure that this isn't silently changing the build output in some way.

Reply:
Should be fine; we were previously also using some of the SYSTEM stuff added in 3.25, but seems not currently used.

Reply:
run UAT is not enough?
I know it build and connect to the cloud with the proposed changes.
What should I look for?